.github/workflows/linting.yml

```
@@ -160,3 +160,19 @@ jobs:
           skip: ./src/control/vendor,./src/control/go.sum,./.git
           ignore_words_file: ci/codespell.ignores
           builtin: clear,rare,informal,names,en-GB_to_en-US
+
+  linting-summary:
+    name: Linting Summary
+    runs-on: ubuntu-22.04
+    needs: [isort, shell-check, log-check, ftest-tags, flake8-lint, doxygen, pylint, codespell]
+    if: always()
```

This creates an un-cancellable job.

```suggestion
    if: always() && !cancelled()
```

https://docs.github.com/en/actions/managing-workflow-runs/canceling-a-workflow#steps-github-takes-to-cancel-a-workflow-run for details.